### ISSUE TITLE
can this damage logger passible record every hit of damage?

### ISSUE THREAD

**Issue by yangloies** [Iuke121/NaoParse#5]
Hi , I am TW mabi palyer. I found this logger yesterday. And connect to TW mabi successfully.
but this logger just logger total damage. not show every data about damage?
https://imgur.com/a/fsYizYr

**Comment by LukasTD** [comment]
That's right, it's more of a classic style DPS meter. I'm surprised it still works, I would have guessed the packet structure or OpCodes to have changed by now. But yeah this is  not a detailed breakdown damage overview like Advanced Combat Tracker. However the code is open source as seen here so you should be able to write something like that if you wished.

**Comment by yangloies** [comment]
thanks I will try ur advice . i will try that ><
